src/test/java/de/tum/cit/aet/artemis/MetricsIntegrationTest.java

```diff
@@ -93,25 +130,56 @@ void shouldReturnExerciseInformation() throws Exception {
             assertThat(exerciseInformation).allSatisfy((id, dto) -> assertThat(id).isEqualTo(dto.id()));
         }
 
-        @Disabled // TODO: reduce jacoco missing by one after enabled
         @Test
         @WithMockUser(username = STUDENT_OF_COURSE, roles = "USER")
-        void shouldReturnAverageScores() throws Exception {
-            // Wait for the scheduler to execute its task
-            participantScoreScheduleService.executeScheduledTasks();
-            await().until(() -> participantScoreScheduleService.isIdle());
+        void shouldReturnCategories() throws Exception {
+            final var result = request.get("/api/metrics/course/" + course.getId() + "/student", HttpStatus.OK, StudentMetricsDTO.class);
+            assertThat(result).isNotNull();
```

Comment:
isnt this check redundant if we check in the following line if a attribute of this element is null? Like if result is null then result.exerciseMetrics will also be null.